apoc/_object_classifier.py

```diff
@@ -180,14 +174,14 @@ def _select_features(self, all_features, features_to_select, labels, ground_trut
 
         Returns
         -------
-        result:list[vector]
+        result:Dict[str, np.ndarray]
             list of vectors corresponding to the requested features. The vectors are shaped (n+1) for n labels. The
             first element corresponds to background.
         ground_truth: ndimage
             selected elements of provided ground truth where it's not 0
```

Comment:
From my testing, I think these vectors are actually length n. Is that correct?

Reply:
(I also need to update the description of the type in the docstring)

Reply:
Yes, all looks good! The docstring was wrong.